### ISSUE TITLE
Iter 4: Store the email template in the database

### ISSUE THREAD

**Issue by cycomachead** [beautyjoy/BJC-Teacher-Tracker#51]
Email templates should be editable by application admins. 

* a single route to edit emails / can just list the templates in the database
* We only really need an index/edit page, creation can happen manually.
* simple formatting for variable replacement. We should be able to insert teacher names / properties.
* The Edit view will would be great if it could show a preview... but it doesn't need to.
* The email templates should be created via application seeds so they exist in development.

Basically should follow this:
https://medium.com/@pratikbh06/customisable-email-templates-in-rails-made-easy-using-panoramic-and-liquid-rails-d1d71f61387d
